alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -85,9 +95,9 @@ class AlertEngineImplTest
 
   test("get active alert") {
     val alertRule = TestObjectGenerator.alertRule(query = "category: ERROR",
-                                                  retryInterval = 1,
-                                                  resultThreshold = Some(1),
-                                                  alertProfiles = List("test@phdata.io"))
+      retryInterval = 1,
```

Comment:
Did you run `sbt scalafmt` before checking this in? These should have already been formatted. I'm wondering if this change was caused by the Intellij formatting.

Reply:
Weird.  I didn't touch this line.  I ran `make package` before commit.  I just ran `sbt scalafmt` and it stayed the same.